yarn.lock

```diff
@@ -3368,6 +3368,20 @@
     global "^4.3.2"
     util-deprecate "^1.0.2"
 
+"@storybook/addons@6.0.0-alpha.15":
```

Comment:
With your changes `yarn.lock` should not be modified and especially not with ref to some `6.0.0-alpha.15` version, can you run `yarn bootstrap --core` to regenerate this file?

Reply:
Thanks, I somehow missed this step!